api/app/views/spree/api/variants/small.v1.rabl

```diff
@@ -14,4 +14,4 @@ child :option_values => :option_values do
   attributes *option_value_attributes
 end
 
-child(:images => :images) { extends "spree/api/images/show" }
+child(:display_images => :images) { extends "spree/api/images/show" }
```

Comment:
@athal7 and I looked at this together and the variant image rules do have an unfortunate side-effect of introducing an N+1 since it looks up images for each rule rather than loading them all up front.

In my opinion the performance impact is very small. With a small set of rules the difference becomes negligible. Weighing the benefits of the new feature versus the performance impact it has I think it's worth the small performance hit.